website/src/components/wordpress.tsx

```diff
@@ -53,13 +64,22 @@ const parseOptions: HTMLReactParserOptions = {
     if ("data" in node) {
       const segments = node.data.match(style)?.filter((x) => x !== undefined)
       if (segments) {
-        console.log(segments)
-
         if (segments?.length === 4) {
-          return pullWords(segments[1]!, segments[2]!, segments[3]!)
+          return (
+            <PullWords
+              slug={segments[1]!}
+              start={parseInt(segments[2]!)}
+              end={parseInt(segments[3]!)}
+            ></PullWords>
+          )
         }
         if (segments?.length === 3) {
-          return pullWords(segments[1]!, segments[2]!)
+          return (
+            <PullWords
+              slug={segments[1]!}
+              start={parseInt(segments[2]!)}
+            ></PullWords>
```

Comment:
I feel like this could be condensed with the previous conditional since everything is the same besides the end argument. However since its only two cases, I think its fine to leave as is if condensing takes too much trouble.